<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkManagementDialog</class>
 <widget class="QDialog" name="NetworkManagementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>564</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>564</width>
    <height>270</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Manage Networks</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/wnt/network.png</normaloff>:/wnt/network.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>225</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_network_button">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>90</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete Network</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/wnt/minus.png</normaloff>:/wnt/minus.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_network_button">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>60</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>View/Edit Network Details</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/wnt/edit.png</normaloff>:/wnt/edit.png</iconset>
   </property>
  </widget>
  <widget class="QTableWidget" name="network_twidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>30</y>
     <width>496</width>
     <height>166</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Network No.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Network Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dimensioning Pressure</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Operating Pressure</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Networks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_network_button">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>30</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Add Network</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/wnt/plus.png</normaloff>:/wnt/plus.png</iconset>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>network_twidget</tabstop>
  <tabstop>add_network_button</tabstop>
  <tabstop>edit_network_button</tabstop>
  <tabstop>delete_network_button</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NetworkManagementDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NetworkManagementDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
